src/components/Component.js

```diff
@@ -16,14 +16,6 @@ export default function Component() {
         })
     }
 
-    const getAirtableRecords = async () => {
```

Comment:
either we can leave this in here for testing or just get rid of Component.js entirely

Reply:
I personally think that it's (for now) fine for testing purposes